### PR TITLE
docs: update documentation to reflect qemu.log behavior

### PR DESCRIPTION
**Pull request by maomarcus**
The documentation now has notes describing the behavior of qemu.log.

Fixes: #2650 

Signed-off-by: Marcus Mao <marcus_mao@yahoo.com>

### CODE REVIEW

**Comment by maomarcus** [comment]
Could someone add the no-backport-needed and forward-port-needed labels?

File: cli/config/configuration-qemu-virtiofs.toml.in [hunk 1]
```diff
@@ -243,6 +243,8 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # This option changes the default hypervisor and kernel parameters
 # to enable debug output where available. This extra output is added
 # to the proxy logs, but only when proxy debug is also enabled.
+# Note: Debug output will only be put into qemu.log in the event
+# of a virtual hardware issue.
```
Comment (devimc) [inline cli/config/configuration-qemu-virtiofs.toml.in:247]: `, otherwise it will be empty` ?  - I'd add this just to be clear

**Review by devimc — COMMENTED** [review]
thanks @maomarcus - just one comment

File: cli/config/configuration-qemu-virtiofs.toml.in (hunk 1 above)
Comment (maomarcus) [reply cli/config/configuration-qemu-virtiofs.toml.in]: added, thanks!

**Comment by devimc** [comment]
thanks @maomarcus - would you mind squashing your commits?

**Comment by maomarcus** [comment]
@devimc sorry just squashed the commits

**Comment by devimc** [comment]
thanks @maomarcus - I think travis is down and the CI was moved to github actions, could you rebase this ?

**Review by fidencio — APPROVED** [review]
lgtm, thanks @maomarcus!

**Review by devimc — APPROVED** [review]

**Pull request merged**